ui/components/app/confirm/info/row/row.tsx

```diff
@@ -34,6 +35,7 @@ export type ConfirmInfoRowProps = {
   tooltip?: string;
   variant?: ConfirmInfoRowVariant;
   style?: React.CSSProperties;
+  labelEndChildren?: React.ReactNode;
```

Comment:
It is safe to assume a `children` property appends something, so should we just call this `labelChildren`?

Reply:
My idea was to give a sense of where it will append at the start or the end, but I agree it is clear just call `labelChildren`